### PR TITLE
Fix broken UI due to refactoring - Closes #3452

### LINKED ISSUE #3452: Fix broken UI due to refactoring
### Description
After refactoring several modules and API utilities to adapt to Lisk SDK v5 endpoints, some screens are broken and need to be rewired. Namely the following:
- Wallet
- Network
- Transactions
- Delegates

### Acceptance Criteria
All screens mentioned above must work as they did before we refactored.

### PR DESCRIPTION
**Pull request by reyraa**
### What was the problem?
This PR resolves #3452

### How was it solved?
- [x] Dashboard screen
- [x] Wallet screen: Transactions tab
- [x] Wallet screen: Votes tab
- [x] Wallet screen: Delegate profile tab -> The API doesn't return `missedBlocks`, `productivity` and `producedBlocks`. We need to discuss further actions.
- [x] Explorer screen: Breaks when user is not a delegate (ej: lskdwsyfmcko6mcd357446yatromr9vzgu7eb8y99)
- [x] Transaction details screen
- [x] Transactions monitor: Transactions list
- [ ] Transactions monitor: Overview -> Broken API, doesn't return enough items.
- [x] Delegates screen: In round tab
- [x] Delegates screen: Outside round tab
- [x] Delegates screen: Votes tab
- [ ] Delegates screen: Sanctioned tab -> There was no sanctioned delegate. I just matched with other tabs.
- [x] Delegates screen: Watch list tab
- [ ] Network screen -> Broken API, doesn't return peer.location.
- [x] Sign message
- [x] Verify message screen

### What does it NOT do
It doesn't update unit tests

### How was it tested?
Visually

### CODE REVIEW

File: src/components/screens/wallet/index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ const Wallet = ({ t, history }) => {
   const activeToken = useSelector(selectActiveToken);
   const { discreetMode } = useSelector(selectSettings);
   const { confirmed, pending } = useSelector(selectTransactions);
+  const isDelegate = !!account.info[activeToken].dpos?.delegate;
```
Comment (isalga) [inline src/components/screens/wallet/index.js:26]: Same

File: src/components/screens/wallet/explorer.js [hunk 2]
```diff
@@ -19,6 +19,7 @@ const Wallet = ({
 }) => {
   const activeToken = useSelector(selectActiveToken);
   const { discreetMode } = useSelector(selectSettings);
+  const isDelegate = !!account.data.dpos?.delegate;
```
Comment (isalga) [inline src/components/screens/wallet/explorer.js:22]: Please use account.info[activeToken].summary.isDelegate. Delegate object will exist with empty information if isDelegate is false

**Review by isalga — CHANGES_REQUESTED** [review]

File: src/components/screens/wallet/index.js (hunk 1 above)
Comment (reyraa) [reply src/components/screens/wallet/index.js:26]: Done

File: src/components/screens/wallet/explorer.js (hunk 2 above)
Comment (reyraa) [reply src/components/screens/wallet/explorer.js:22]: Done

File: src/components/screens/wallet/delegateProfile/delegateProfile.js [hunk 3]
```diff
@@ -23,41 +23,45 @@ import PerformanceView from './performanceView';
...
   useEffect(() => {
-    delegate.loadData();
     voters.loadData();
-  }, [address]);
+  }, [account]);
 
   useEffect(() => {
-    lastBlockForged.loadData({ height: delegate.data.dpos?.delegate?.lastForgedHeight });
-  }, [delegate.data.dpos?.delegate?.lastForgedHeight]);
+    lastBlockForged.loadData({ height: delegate?.lastForgedHeight });
```
Comment (UsamaHameed) [inline src/components/screens/wallet/delegateProfile/delegateProfile.js:36]: Here, we should not null check the `delegate` variable before making the request. Can you please move the null check before the api call? 

```suggestion
	if(delegate?.lastForgedHeight) {
		lastBlockForged.loadData({ height: delegate.lastForgedHeight });
	}
```

File: src/components/screens/wallet/delegateProfile/index.js [hunk 4]
```diff
@@ -13,20 +13,11 @@ const mapStateToProps = state => ({
...
-    transformResponse: response => response.data[0],
-  },
   voters: {
     apiUtil: (network, params) => getVoters({ network, params }),
     defaultData: [],
-    getApiParams: (_, ownProps) => ({
-      address: ownProps.address,
-    }),
+    getApiParams: (_, ownProps) => ({ address: ownProps.account.summary.address }),
+    transformResponse: response => (response.data.votes ? response.data.votes : []),
```
Comment (UsamaHameed) [inline src/components/screens/wallet/delegateProfile/index.js:20]:
```suggestion
    transformResponse: response => (response.data.votes ?? []),
```

**Review by UsamaHameed — CHANGES_REQUESTED** [review]

**Review by isalga — CHANGES_REQUESTED** [review]

File: src/components/screens/monitor/transactions/index.js [hunk 5]
```diff
@@ -6,14 +6,30 @@ import { transformStringDateToUnixTimestamp } from '@utils/datetime';
 import { toRawLsk } from '@utils/lsk';
 import Transactions from './transactions';
 
+const defaultTimestampRange = '1483228800000:1735689600000';
+const defaultAmountRange = '0:9223372036854775807';
```
Comment (UsamaHameed) [inline src/components/screens/monitor/transactions/index.js:10]: @reyraa would be really helpful if we can add comments as to why these particular values have been chosen as defaults.

**Review by UsamaHameed — APPROVED** [review]

**Pull request merged**